docs/architecture/stack/evm-state-manager/index.mdx

```diff
@@ -1,31 +1,253 @@
+data which it encoded. If you have the hash of the root—the only node without a parent—you can 
+theoretically derive from it the data of any node in the entire tree. 
+
+As a layer 2 (L2) network, Linea is in the business of making transacting faster and more efficient.
+Linea implements a sparse Merkle tree to track account state and generate and store proofs, and 
+unlock greater efficiency when compared to standard Merkle trees, which require recomputation for 
+every block, leading to excessive computational demands.
+
+### Sparse Merkle trees
+
+Linea's state management uses sparse Merkle trees to minimize computation and contribute to the 
+blockchain's efficiency. 
+
+A sparse Merkle tree is a variation of a standard Merkle tree where not all leaf nodes are filled 
+with data; instead, data is only stored in nodes where it's needed. It is a complete tree of fixed 
+depth, meaning that all branches of the tree have the same length—i.e. the same number of leaves. 
+
+At initialization—the beginning of the chain's history—all leaf nodes are set to a default value, 
+which is typically a hash of a specific value, such as zero. Because all leaf nodes have the same 
+hash value, the parent nodes and higher-level nodes also have the same hash value. A node whose 
+hash is the default value for its level is therefore considered to represent an _empty_ subtree. 
+
+<div class="center-container">
+  <div class="mermaid-medium">
+    ```mermaid
+    flowchart TD
+        A[root] --> B["`**node A**`"
+        null] & C["`**node B**`"
+        contains data]
+        B --> D["`**node C**`":
+        null] & E["`**node D:**`"
+        null]
+        C --> F["`**node E**`":
+        data] & G["`**node F**`":
+        data]
+    ```
+  </div>
+</div>
+
+In the example above, the children of node A (leaves) contain null values, which means node A does 
+too. Node B, meanwhile, reflects that its children also contain values.
+
+With this construction, we do not need to keep track of every individual node's hash. Instead, we 
+can assume hashes that reflect the default value are empty, and the subtree or node that lies 
+further down the chain of nodes can be disregarded; we only need to pay attention to the ones that 
+correspond to _non-empty_ subtrees.
+
+## Cryptographic accumulator
+
+In this context, we can consider Linea's sparse Merkle tree as a type of "cryptographic 
+accumulator". A cryptographic accumulator is a type of cryptographic primitive encoding a collection 
+of items into very short strings and allowing read/write operations to be proven. Merkle trees and 
+sparse Merkle trees are elementary examples of accumulators but there are others with more powerful 
+capabilities.
+
+Linea's state manager uses an extended version of a sparse Merkle tree that enables it to prove all 
+CRUD (create, read, update, delete) operations for a key-addressed database. As an outline, the 
```

Comment:
Did this go through SME review? I am still confused by the CRUD statement here. Thought I added comment on this before.
My (probably naive) takeaway from A's doc is that the ability to overwrite the 0 is a feature required by the proof generator --> I don't think this update extends to storing state. I would recommend having SME review this as any mention of UD (especially the delete) must be made clear --> the immutable nature of blockchain (as in the typical use of PMT) allows for old data to be ignored thanks to availability of more recent data/updated state, this is NOT the same as a typical database's CRUD behaviour. FAO @jlwllmr

Reply:
@m4sterbunny thank you — as discussed, following up with SME on this issue 🫡